discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -515,3 +529,72 @@ def wait_for_cluster_to_be_in_installing_status(self):
             cluster_id=self.id,
             statuses=[consts.ClusterStatus.INSTALLING]
         )
+
+    def reset_cluster_and_wait_for_ready(self, cluster, nodes):
+        # Reset cluster install
+        cluster.reset_install()
+        assert cluster.is_in_insufficient_status()
+        # Reboot required nodes into ISO
+        cluster.reboot_required_nodes_into_iso_after_reset(nodes=nodes)
+        # Wait for hosts to be rediscovered
+        cluster.wait_until_hosts_are_discovered()
+        cluster.wait_for_ready_to_install()
+
+    def verify_no_logs_uploaded(self, cluster, path):
+        with pytest.raises(ApiException) as ex:
+            cluster.download_installation_logs(path)
+        assert "No log files" in str(ex.value)
+
+    def verify_logs_uploaded(self, path, expected_min_log_num, installation_success):
```

Comment:
Please change `path` to `cluster_tar_path`

Reply:
done